tests/test_backend.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import requests
+import time
+import turbopuffer as tpuf
+from turbopuffer import backend as tpuf_backend
+from unittest import mock
+
+
+def mock_response_returning(status_code, reason):
+    response = mock.Mock(spec=requests.Response)
+    response.status_code = status_code
+    # None of these values really matter, they're just accessed in Backend, so
+    # they need to exist.
+    response.url = tpuf.api_base_url
+    response.reason = reason
+    response.headers = {}
+    response.request = mock.Mock(spec=requests.PreparedRequest)
+    response.request.url = response.url
+    response.raise_for_status = lambda: requests.Response.raise_for_status(response)
+    return response
+
+
+@pytest.mark.xdist_group(name="group1")
```

Comment:
We only need to set the group for tests that need to run sequentially within the group.

pytest will run any function that starts with `test_`, no annotation needed

Reply:
And all the other tests have them because they're accessing a shared namespace in prod.  Alright, my bad for the copy/paste.